platforms/corda-ent/configuration/roles/setup/nmap/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+##############################################################################################
+# This role creates the value file for nmap
+##############################################################################################
+
+# create deployment file for CENM nmap service
+- name: "create nmap"
+  include_role:
+    name: helm_component
+  vars:
+    component_name: "{{ org.name | lower }}nmap"
+    component_ns: "{{ org.name | lower }}-ent"
+    type: "nmap"
+    values_dir: "{{playbook_dir}}/../../../{{ org.gitops.release_dir }}"
+    name: "{{ org.name | lower }}"
+    charts_dir: "{{ org.gitops.chart_source }}"
+    notary_name: "{{ org.services.notary.name }}"
+    idman_name: "{{ org.services.idman.name }}"
+    git_url: "{{ org.gitops.git_ssh }}"
+    git_branch: "{{ org.gitops.branch }}"
+    node_name: "{{ org.services.networkmap.name }}"
+    storageclass: "awsstorageclass"
+    init_container_name: "{{ network.docker.url }}/alpine-utils:1.0"
+    image_pull_secret: "regcred"
+    nmap_image_name: corda/enterprise-networkmap
```

Comment:
image name and tag should be mapped as per network.version as is done for opensource Corda

Reply:
yep changing this